cmd/kured/main.go

```diff
@@ -90,6 +93,11 @@ func main() {
 	rootCmd.PersistentFlags().StringVar(&slackChannel, "slack-channel", "",
 		"slack channel for reboot notfications")
 
+	rootCmd.PersistentFlags().StringVar(&notifyURL, "notify-url", "",
```

Comment:
Why do you need to expose this?
Isn't it enough to have slack-hook-url, username, and channel?

If you still want to have this notify-url part of this commit, I suggest that you handle the case where both slack-* arguments and that notify-url is provided.

Reply:
Removed.

Reply:
You didn't need to _remove_ the notifyURL. For me that's perfectly fine to expose the generic mechanism in a commit on this PR.